### PR TITLE
Remove post comments

### PR DESCRIPTION
**Pull request by pnbabu**
This PR removes the parsing of post comments from NESTML models. As a consequence, only pre and inline comments will be parsed.

This PR is a pre-requisite for #866

### CODE REVIEW

**Comment by clinssen** [comment]
Thank you! Could you update the "NESTML language concepts" documentation page as well?

File: doc/nestml_language/nestml_language_concepts.rst [hunk 1]
```diff
@@ -339,15 +340,17 @@ If a comment shall be attached to an element, no white lines are allowed.
 
 .. code-block:: nestml
 
-   V_m mV = -55 mV # I am a comment of the membrane potential
+   # I am not a comment of the membrane potential.
+
    # I am a comment of the membrane potential.
```
Comment (clinssen) [inline doc/nestml_language/nestml_language_concepts.rst:345]:
```suggestion
```

File: doc/nestml_language/nestml_language_concepts.rst [hunk 2]
```diff
@@ -327,10 +327,11 @@ Example of single or multi-line comments:
    #  a comment
    #   over several lines.
 
-To enable NESTML to recognize which element a comment belongs to, the following approach has to be used: there should be no white line separating the comment and its target. For example:
+To enable NESTML to recognize which element a comment belongs to, the following approach has to be used: there should be no white line separating the comment and its target and the comment should be placed before the target line or on the same line as the target. For example:
 
 .. code-block:: nestml
 
+   # I am a comment of the membrane potential
    V_m mV = -55 mV # I am a comment of the membrane potential
```
Comment (clinssen) [inline doc/nestml_language/nestml_language_concepts.rst:336]:
```suggestion
```

File: doc/nestml_language/nestml_language_concepts.rst [hunk 3]
```diff
@@ -339,15 +340,17 @@ If a comment shall be attached to an element, no white lines are allowed.
 
 .. code-block:: nestml
 
-   V_m mV = -55 mV # I am a comment of the membrane potential
+   # I am not a comment of the membrane potential.
+
    # I am a comment of the membrane potential.
+   V_m mV = -55 mV # I am a comment of the membrane potential
 
 Whitelines are therefore used to separate comment targets:
```
Comment (clinssen) [inline doc/nestml_language/nestml_language_concepts.rst:348]: Remove this line and the following code block (redundant with previous text).

File: doc/nestml_language/nestml_language_concepts.rst (hunk 1 above)
Comment (pnbabu) [reply doc/nestml_language/nestml_language_concepts.rst:345]: Why do you suggest to remove this?

Comment (clinssen) [reply doc/nestml_language/nestml_language_concepts.rst:345]: to illustrate that if a whiteline separates a pre comment and its correponding element, it is not picked up as belonging to that element (this is the way it is, right?)

Comment (pnbabu) [reply doc/nestml_language/nestml_language_concepts.rst:345]: Yes, you are right. That is why I have a pre-comment attached to that element (on L345) and one more comment before that separated by whitespace (on L343). I think your suggestion points to removing the pre-comment on L345. Would you still like for me to remove that?

**Review by clinssen — APPROVED** [review]

**Comment by clinssen** [comment]
Many thanks!

**Pull request merged**